WordPress/wporg-developer
# Update homepage layout

Closes #213 

Depends on https://github.com/WordPress/wporg-parent-2021/pull/108 for Card list styles.

Latest News section will addressed in another iteration when https://github.com/WordPress/wporg-mu-plugins/issues/448 is done.

### Screenshots

| Desktop | Tablet | Mobile |
|-|-|-|
| ![localhost_8888_(Desktop) (8)](https://github.com/WordPress/wporg-developer/assets/1017872/3f02291f-9c70-4b1d-b3a4-cea15523e70b) | ![localhost_8888_(iPad) (1)](https://github.com/WordPress/wporg-developer/assets/1017872/a8aca808-f168-411d-a957-f92a9126e758) | ![localhost_8888_(Samsung Galaxy S20 Ultra) (11)](https://github.com/WordPress/wporg-developer/assets/1017872/b840417a-f5ed-4f3b-9736-3a948c5da8a2) |

### Testing

1. Run the [Card style list branch](https://github.com/WordPress/wporg-parent-2021/pull/108) for the parent theme
2. Load the homepage and check responsive styles and interactive states for the cards
